src/child/proptypes/favourites.js

```diff
@@ -0,0 +1,8 @@
+import { PropTypes } from 'react';
```

Comment:
Could rename the `proptypes` directory to `propTypesShapes` - I think camelCase will be more consistent and I don't think I'd want to encourage any other propTypes going in here, only shapes which are reused.